### PR TITLE
Fix boolean argument in Nair Lauritzen divergent

### PR DESCRIPTION
**Pull request by ta440**
Set the boolean argparse option of --background_flow to default to False. This means that both false and true (--background_flow=True) configurations can be run, whereas previously it became True irrespective of command line input.

### CODE REVIEW

File: case_studies/transport/nair_lauritzen_divergent.py [hunk 1]
```diff
@@ -61,6 +61,8 @@ def nair_lauritzen_divergent(
     # Our settings for this set up
     # ------------------------------------------------------------------------ #
 
+    print(no_background_flow)
```
Comment (tommbendall) [inline case_studies/transport/nair_lauritzen_divergent.py:64]: Can you remove this?

**Review by tommbendall — CHANGES_REQUESTED** [review]
Thanks Tim for spotting that this wasn't working.

Two main things:
- can you apply the same change to the non-divergent test?
- can you update the calls in the `test_transport.py` file?

File: case_studies/transport/nair_lauritzen_divergent.py (hunk 1 above)
Comment (ta440) [reply case_studies/transport/nair_lauritzen_divergent.py:64]: Oops, I have a habit of doing this!

**Review by tommbendall — APPROVED** [review]
Thanks Tim!

**Pull request merged**